server/status.go

```diff
@@ -87,23 +86,16 @@ func (s *statusServer) registerHandlers(mux *http.ServeMux) {
 	mux.HandleFunc(statusTransactionsKeyPrefix, s.handleTransactionStatus)
 }
 
-// marshalJSON marshals the provided obj into indented JSON format.
-func (s *statusServer) marshalJSON(r *http.Request, obj interface{}) ([]byte, error) {
-	return json.MarshalIndent(obj, "", "  ")
-}
-
 // handleStatus handles GET requests for cluster status.
 func (s *statusServer) handleStatus(w http.ResponseWriter, r *http.Request) {
-	w.Header().Set("Content-Type", "application/json")
-
 	cluster := &status.Cluster{}
-
-	b, err := s.marshalJSON(r, cluster)
+	b, contentType, err := util.MarshalResponse(r, cluster, util.AllEncodings)
```

Comment:
We shouldn't pass AllEncodings here, because it may try to encode the message as a protobuf if the request includes an appropriate Accept header (this will panic because the message is not a protobuf type). We should explicitly limit it to `[]util.EncodingType{util.JSONEncoding}`.